docs/api.rst

```diff
@@ -65,6 +65,16 @@ Writing to other formats.
    AnnData.write_zarr
 
 
+Experimental development API
+----------------------------
+
+.. autosummary::
+   :toctree: .
+
+   dev.multi_files.AnnDataSet
```

Comment:
I'd suggest to have both objects exported at the root level of `anndata.dev`. The submodules are confusing.

Reply:
Done.